skt/__init__.py

```diff
@@ -405,11 +405,19 @@ def merge_patchwork_patch(self, uri):
                           patchinfo.get("name").replace(',', ';')))
 
     def merge_patch_file(self, path):
+        if not os.path.isfile(path):
+            raise Exception("Patch %s not found" % os.path.basename(path))
+        abspath = os.path.abspath(path)
         try:
-            self.git_cmd("am", path)
-        except subprocess.CalledProcessError:
+            subprocess.check_output(["git", "am", abspath], cwd=self.wdir)
```

Comment:
Can you also add the `env` locale override (as you can see in the other function)?

Reply:
Ah what does it mean `env`? I see it in several `subprocess.Open`. But I am not sure about it

Reply:
`env` overrides process' environment. You can read more about what environment variables are for example [here](https://wiki.archlinux.org/index.php/environment_variables). The variable we add ensures git's output stays same (in English) and independent from user's language or charset settings, so we can easily search it and add to reports.